Беспутин Иван

PRIZRAKeee opened the pull request:
[Посмотреть решение](https://hruview.herokuapp.com/PRIZRAKeee/markup-task-6/master/index.html)

PRIZRAKeee committed:
Create folder images

PRIZRAKeee committed:
Add images

PRIZRAKeee committed:
Update inde.css

PRIZRAKeee committed:
Update css 2

PRIZRAKeee committed:
Update html

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

PRIZRAKeee commented:
Я не успел сделать немного фильтр и красоту навести с котиками. Но я доделаю это

PRIZRAKeee committed:
Update css 3

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

PRIZRAKeee committed:
Update CSS 4

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

dimastark commented:
Оке, буду ждать.

PRIZRAKeee committed:
Update CSS 5

PRIZRAKeee committed:
Update HTML 2

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

PRIZRAKeee commented:
Я готов к проверке

dimastark commented on images/txt.txt:
```diff
@@ -0,0 +1 @@
+
```
🤔

PRIZRAKeee replied:
Делал это для того, чтобы создать папку. Научился закидывать файлы на github, больше так не буду

dimastark commented on index.css:
```diff
@@ -0,0 +1,477 @@
+
+    .item-cat
+    {
+        display: block !important;
```
А почему?

PRIZRAKeee replied:
Тут сделал important чтобы не случилось следующей ситуации: когда в фильтре на десктопе выбрана какая-нибудь порода, а после мы уменьшаем размер экрана до планшета или мобильной версии, то остальные котики не появятся. Установил important, чтобы начиная с планшетной версии отображались все котики.

dimastark commented on index.css:
```diff
@@ -0,0 +1,477 @@
+
+.container-cats
+{
+    width: calc( 100% - 250px);
```
Пробельчик)

PRIZRAKeee replied:
исправлюсь

dimastark commented on index.css:
```diff
@@ -0,0 +1,477 @@
+body
```
А где grid? Может заюзаешь где-нибудь?

PRIZRAKeee replied:
Получилось сделать без grid. Обязательно применю их при выполнении 7 задания.

dimastark commented on index.html:
```diff
@@ -3,7 +3,310 @@
+    <link rel="stylesheet" href="index.css">
 </head>
-<body>
+<body class="body">
```
Ну уж тут то можно и без класса.

PRIZRAKeee replied:
Исправлюсь

dimastark commented on index.html:
```diff
@@ -3,7 +3,310 @@
+        <input type="radio" id="breed3" class="breed3 breed__input" name="cats">
+        <input type="radio" id="breed4" class="breed4 breed__input" name="cats">
+        <input type="radio" id="breed5" class="breed5 breed__input" name="cats">
+        <section class="container-cats">
```
`cats-container`?

dimastark commented on index.html:
```diff
@@ -3,7 +3,310 @@
+                        title="Фотография кота Леонардо" 
+                        class="item-cat__photo">
+                    <div class="item-cat__description">
+                        <p class="item-cat__p item-cat__p_name">Леонардо</p>
```
Может придумать более осмысленное имя класса?

dimastark commented on index.html:
```diff
@@ -3,7 +3,310 @@
+                </section>
+                <section class="item-cat__information-about-cat">
+                    <p class="item-cat__p">Дружелюбный, ласковый, ручной</p>
+                    <p class="item-cat__p item-cat__p_margin">
```
_модификатор обычно имеет имя не про стиль, а про "модификацию" - `active`, `highlighted` – ну в общем про новое состояние.

dimastark reviewed (commented):
* В планшетном и десктопном режиме футер не работает
* Фильтры не совсем правильные - можно отфильтровать только по одной категории, удобнее было бы иметь возможность выбирать несколько

Десктопная версия шикарна! Молодец 🍏

PRIZRAKeee committed:
Доработка 1

Добавил возможность выбора нескольких пород в фильтре. Изменил название фильтра "Нет" на более осмысленный "Все".

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

PRIZRAKeee committed:
Доработка 2

Убрал фиксированное положение футера

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

PRIZRAKeee commented:
А в 6 задании для вёрстки есть дополнительные баллы? У некоторых студентов есть оценки extra

PRIZRAKeee committed:
Доработка 3

Постарался дать более осмысленные имена классам и исправил небольшие ошибки

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

PRIZRAKeee commented:
Готов к проверке

PRIZRAKeee committed:
Доработка 4

Исправил отступ снизу элемента main

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

dimastark commented:
> А в 6 задании для вёрстки есть дополнительные баллы? У некоторых студентов есть оценки extra

Нет, похоже кто-то добавил по ошибке.

dimastark commented:
🍏 
Выглядит отлично!
